index.html

```diff
@@ -13,32 +13,37 @@
     <!-- Tailwind core CSS -->
     <script src="https://cdn.tailwindcss.com"></script>
     <link href="./dist/styles.css" rel="stylesheet" />
+
+    <!-- Poppins font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
   </head>
 
-  <body>
+  <body class="font-poppins">
     <section class="flex flex-wrap mx-auto max-w-[500px] md:max-w-[752px] lg:max-w-[1023px]">
-      <div class="border border-gray-300 rounded-lg m-2 lg:max-w-[325px] md:max-w-[360px] max-w-[500px] hover:shadow-lg">
+      <div class="card">
         <img src="https://picsum.photos/500/300?v=1" alt="random image" class="rounded-t-lg">
-        <p class="p-4">The aim of this challenge is to put into practice how to use Tailwind to create a responsive UI. Before adding custom styles, check the docs to see if these styles can be achieved using a Tailwind class.</p>
+        <p class="m-4 line-clamp-2">The aim of this challenge is to put into practice how to use Tailwind to create a responsive UI. Before adding custom styles, check the docs to see if these styles can be achieved using a Tailwind class.</p>
         <div class="flex justify-between items-center px-4 pb-4">
           <p class="text-gray-500">8 mins</p>
-          <button type="button" class="font-bold px-4 py-2 mr-2 bg-red-400 text-white rounded-xl hover:bg-red-800 focus:bg-red-800 focus:border-2 focus:border-red-300">Read more</button>
+          <button type="button" class="btn">Read more</button>
```

Comment:
As a good practice, when defining custom class names (in this case your `btn`), try to use more specific names in order to avoid using a name already defined.